Formula/cassandra-cpp-driver.rb

```diff
@@ -0,0 +1,37 @@
+class CassandraCppDriver < Formula
+  desc "DataStax C/C++ Driver for Apache Cassandra"
+  homepage "https://docs.datastax.com/en/developer/cpp-driver/latest"
+  url "https://github.com/datastax/cpp-driver/archive/2.15.0.tar.gz"
+  sha256 "8cccffc7f3b7a8edc51c5c4344fd6a60d5a7beec27a1a3fd52742003a2675cb9"
+  head "https://github.com/datastax/cpp-driver.git"
+
+  depends_on "autoconf" => :build
+  depends_on "automake" => :build
```

Comment:
Why in the world does this formula being built with CMake depend on autoconf, automake and libtool??!?

Reply:
Looking at the [documentation](https://docs.datastax.com/en/developer/cpp-driver/2.15/topics/building/#dependencies) it appears there is no need to add all this auto-tools stuff...